nntrainer/tensor/memory_pool.h

```diff
@@ -27,6 +27,41 @@
 #include <memory_planner.h>
 #include <tensor_wrap_specs.h>
 
+#include <cstdlib>
+#include <dynamic_library_loader.h>
+#include <functional>
+#include <memory>
+#include <vector>
+#if defined(_WIN32)
+#ifdef max
+#undef max
+#undef min
+#endif
+#define NOMINMAX
+#define O_SYNC 0UL
+#include <io.h>
+#include <sysinfoapi.h>
+#include <windows.h>
+#else
+#include <sys/mman.h>
+#include <unistd.h>
+#endif
+
+#include <engine.h>
+#include <iostream>
+#include <mem_allocator.h>
+#include <set>
+
+static const std::string func_tag = "[MemoryPool] ";
+typedef void *(*RpcMemAllocFn_t)(int, uint32_t, int);
+typedef void (*RpcMemFreeFn_t)(void *);
+
+enum {
+  DL_NOW = 0x0001,
+  DL_LOCAL = 0x0002,
+  DL_GLOBAL = 0x0004,
+};
```

Comment:
This function type and enum will be used only when `__ANDROID__` tag is defined in qualcomm's toolchain.
or Please think of puttuing the condition on importing headers as well.

```suggestion
#if defined(__ANDROID__)
typedef void *(*RpcMemAllocFn_t)(int, uint32_t, int);
typedef void (*RpcMemFreeFn_t)(void *);

enum {
  DL_NOW = 0x0001,
  DL_LOCAL = 0x0002,
  DL_GLOBAL = 0x0004,
};
#endif
```